cmd/install/install.go

```diff
@@ -608,6 +609,19 @@ func hyperShiftOperatorManifests(opts Options) ([]crclient.Object, []crclient.Ob
 	objects = append(objects, trustedCABundle)
 
 	if len(opts.ExternalDNSProvider) > 0 {
+		// Setting the proxy for external-dns is best-effort, ignore errors
```

Comment:
What error is being ignored?

Reply:
Any failure to get the Proxy cluster config.  No/bad kubeconfig, kubeconfig to non-OCP cluster, etc.